dataactcore/models/validationModels.py

```diff
@@ -31,7 +31,6 @@ class RuleType(Base):
     description = Column(Text)
 
     TYPE_DICT = None
-    TYPE_LIST = ["TYPE", "EQUAL","NOT EQUAL","LESS","GREATER","LENGTH","IN_SET","SUM"]
```

Comment:
These were not being used and the information they held was obsolete anyway.  We cleaned up some of these a while ago, but these two were missed, so I'm removing them here.  Just found another one in JobType, I'm pushing a change to delete that as well.